lib/shopify_api/resources/fulfillment_request.rb

```diff
@@ -0,0 +1,11 @@
+module ShopifyAPI
+  class FulfillmentRequest < Base
+    init_prefix :order
+
+    def order_id
+      @prefix_options[:order_id]
+    end
+
+    def cancel; load_attributes_from_response(post(:cancel, {}, only_id)); end
```

Comment:
So to clarify we don't want to add the ability to cancel a fulfillment from here? Does cancelling a fulfillment request cancel the associated fulfillment (in Shopify)?